File: problemas/iniciante/1134/1134.lua
alcool = 0
gasolina = 0
diesel = 0
while true do
  codigo = tonumber(io.read(), 10)

  if codigo == 1 then
    alcool = alcool + 1
  else
    if codigo == 2 then
      gasolina = gasolina + 1
    else
      if codigo == 3 then
        diesel = diesel + 1
      else
        if codigo == 4 then
          break
        end
      end
    end
  end
end

print("MUITO OBRIGADO")
print(string.format("Alcool: %d", alcool))
print(string.format("Gasolina: %d", gasolina))
print(string.format("Diesel: %d", diesel))

File: problemas/iniciante/1134/1134.kt
import java.util.*

fun main(args: Array<String>) {
  val sc = Scanner(System.`in`)

  var alcool = 0
  var gasolina = 0
  var diesel = 0
  while (true) {
    val codigo: Int = sc.nextInt()

    if (codigo == 1) {
      alcool += 1
    } else if (codigo == 2) {
      gasolina += 1
    } else if (codigo == 3) {
      diesel += 1
    } else if (codigo == 4) {
      break
    }
  }

  println("MUITO OBRIGADO")
  println(String.format("Alcool: %d", alcool))
  println(String.format("Gasolina: %d", gasolina))
  println(String.format("Diesel: %d", diesel))
}
